components/apimgt/org.wso2.carbon.apimgt.impl/src/main/java/org/wso2/carbon/apimgt/impl/utils/APIUtil.java

```diff
@@ -1432,6 +1435,12 @@ public static String createEndpoint(String endpointUrl, Registry registry) throw
             String msg = "Malformed XML found in the External Stores Configuration resource";
             log.error(msg, e);
             throw new APIManagementException(msg, e);
+        } catch (ClassNotFoundException e) {
+            log.error("Requested APIPublisher Class couldn't found", e);
```

Comment:
Previous catch boxes catch and throw APIManagementException. Any reason just only logging the exception?